OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/AlertDialogFragment.java

```diff
@@ -29,6 +38,23 @@ public static AlertDialogFragment newInstance(@Nullable String title, @NonNull S
         return fragment;
     }
 
+    /**
+     * Creates and shows a new instance of this dialog fragment with the provided title and message.
+     *
+     * @param title   The title to set.
+     * @param message The message to set.
```

Comment:
You can remove the "with the provided title and message" since that's obvious from the parameter signature immediately afterwards. You can also remove the "to set" from there, which is redundant.